### ISSUE TITLE
au new --here using webpack `Unexpected token )`

### ISSUE THREAD

**Issue by 4imble** [aurelia/cli#721]
**I'm submitting a bug report**

* **CLI Version:**
0.31.2

**Please tell us about your environment:**
* **Operating System:**
Windows 10

**Current behavior:**
Using au new --here and choosing the following options fails

3) Webpack,
1) Web,
2) Typescript,
1) Default,
2) Less
1) None,
2) No,
1) Visual Studio Code

```
ERROR [ProjectCreation] Failed to create the project due to an error: Unexpected token )
INFO [ProjectCreation] C:\Users\user\AppData\Roaming\npm\node_modules\aurelia-cli\lib\commands\new\buildsystems\webpack\index.js:41
  ).addToContent(
  ^
SyntaxError: Unexpected token )
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at ui.clearScreen.then.then.then.answer (C:\Users\user\AppData\Roaming\npm\node_modules\aurelia-cli\lib\workflow\activities\project-create.js:47:30)
```
**Fix**
There is a stray comma on line 40 that is causing this to fail, i removed it and reran the steps and it all works.

https://github.com/aurelia/cli/blob/053ad98d9dad05ba16de861e57f010d863107d96/lib/commands/new/buildsystems/webpack/index.js#L40

```
ProjectItem.resource('environment.ext', 'tasks/environment.ext', project.model.transpiler),<--HERE 
.addToContent(
```

**Comment by JeroenVinke** [comment]
Sorry about that. If you're getting this problem, use node version 8 until we can get the fix in. Apparently nodejs 8 is the only version that supports trailing comma's

**Comment by 4imble** [comment]
No worries, it's not a blocker for me as I edited the file locally. 
Thought it best I report it in case it was unknown. :thumbsup:

**Comment by JeroenVinke** [comment]
@4imble mind if I change the title to `Unexpected token )` ?

**Comment by JeroenVinke** [comment]
since the fix is in master now, you can also install it that way: `npm install aurelia/cli#master -g` and switch back to the latest after the patch release

**Comment by JeroenVinke** [comment]
Released as 0.32.3
